include/taskolib/Sequence.h

```diff
@@ -112,10 +115,8 @@ class Sequence
      * The label should describe the function of the sequence clearly and concisely.
      * The label must not be empty. Leading and trailing whitespaces will be removed.
      *
-     * \param label descriptive and expressive label.
-     *
-     * \exception Error is thrown if the label is empty or if its length exceeds
-     *            max_label_length bytes.
+     * \param label     descriptive and expressive label.
+     * \exception       is thrown if the label does not fit.
```

Comment:
The hint that `max_label_length` is the maximum length might still be helpful to users.